### PR TITLE
FAQ: how can I cancel pending TX

### PR DESCRIPTION
**Pull request by MarnixCroes**
cc86d83a88887b214fd8c5ebeb6ecaa81e32edbc to improve the title of the image, so it's also correct for cancelling: 
`SpeedUpHistory.png` -> `HistoryRightClick.png` 
`SpeedUpConfirm.png` -> `SpeedUpTxConfirm.png`

### CODE REVIEW

File: docs/FAQ/FAQ-UseWasabi.md [hunk 1]
```diff
@@ -697,17 +697,34 @@ To speed up a transaction:
...
 
 - Enter your Password to send the new transaction.
 - Transaction succesfully speeded up!
 
 Read more [here](/using-wasabi/Send.md#speed-up-or-cancel-transaction).
 
+### How can I cancel a pending/unconfirmed transaction?
+
+A pending (unconfirmed) transaction can be cancelled by using the _Cancel Transaction_ feature which will send a new transaction with a higher fee rate to replace the current one.
+The replacement transaction is sent to an address of the user's wallet, so the bitcoin comes back to the user's wallet and the transaction can be considered "cancelled".
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:714]:
```suggestion
The replacement transaction is sent to an address of the user's wallet, so the bitcoin comes back and the transaction can be considered "cancelled".
```

File: docs/FAQ/FAQ-UseWasabi.md [hunk 2]
```diff
@@ -697,17 +697,34 @@ To speed up a transaction:
...
-![Speed Up History](/SpeedUpHistory.png "Speed Up History")
+![Speed Up Tx History](/HistoryRightClick.png "Speed Up Tx History")
 
 - Confirm you want to pay the additional fee for speeding it up.
 
-![Speed Up Confirm](/SpeedUpConfirm.png "Speed Up Confirm")
+![Speed Up Tx Confirm](/SpeedUpTxConfirm.png "Speed Up Tx Confirm")
 
 - Enter your Password to send the new transaction.
 - Transaction succesfully speeded up!
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:707]: These are obvious steps that can be removed, and to be consistent.

```suggestion
```

**Review by yahiheb — CHANGES_REQUESTED** [review]
Ude `btc` denomination for `CancelTxConfirm.png` instead of sats to be consistent with the other screenshots.

**Review by MaxHillebrand — DISMISSED** [review]

File: docs/FAQ/FAQ-UseWasabi.md (hunk 2 above)
Comment (yahiheb) [reply docs/FAQ/FAQ-UseWasabi.md:707]: Why shouldn't these be removed? And if they shouldn't why not add them to the cancel transaction steps?

**Pull request merged**